spk/tcl/src/installer.sh

```diff
@@ -16,11 +19,10 @@ preinst ()
 postinst ()
 {
 	# Create symlinks
```

Comment:
Add a symlink like this: `ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}`. See other packages for an example.